xls/dslx/bytecode/bytecode_interpreter_test.cc

```diff
@@ -1519,6 +1519,78 @@ fn main() {
   EXPECT_TRUE(value.IsUnit());
 }
 
+// GH 981
```

Comment:
nit: in the absence of well defined convention, let's use the fully qualified url

Reply:
Changed to full URL